README.md

```diff
@@ -17,9 +17,9 @@ Add your answers to this markdown file.
 
 **1. Asymptotic analysis disregards constants, which can make considerable differences, especially in cases of small amounts of data sets.**
 
-**2. Whenever 2 algorithms have the same asymptotic complexity, it seems that they would have the same runtime. This is misleading as that is often untrue, depending on many factors, including the distribution of the data sets.**
+**2. Some algorithms have a low asymptotic runtime complexity - however, they may be prone to issues such as cache misses or expensive comparisons or operations. Thus, in reality, the algorithm may be a lot slower than the asymptotic analysis would lead someone to believe.**
 
-**3. The actual implementation of the algorithm can substantially impact its performance. Small choices in code as well as the choices of data structures and compiler optimizations.**
+**3. I could simply point out that the machine being used to run the program has a very large impact on the actual performance comparative to the asymptotic analysis. An algorithm being run on a supercomputer would obviously result in an actual performance that is a lot different than if we were to run the algorithm on a tablet.**
```

Comment:
Doesn't that just amount to a constant factor, thus being the same as your first reason?